project/Common.scala

```diff
@@ -28,9 +28,7 @@ object Common extends AutoPlugin {
     "-deprecation",
     "-feature",
     "-unchecked",
-    "-encoding",
-    "utf8",
-    "-target:jvm-1.8",
```

Comment:
Why did you remove `-target:jvm-1.8`?

Reply:
It should have been removed at the time 2.11 support was dropped. It's the default on 2.12 and 2.13 and 3.0 regardless of what JVM you're building on.